### PR TITLE
feat[api]: CRUD for parties, candidates, polling stations, etc. 

### PR DESCRIPTION
**Pull request by Jatewo**
## Description

GET, POST, PATCH, and DELETE endpoints for parties, candidates, polling stations, nomination districts, and constituencies, test setup, cors settings, linting and more.

## Changes made
- CRUD endpoints for various objects
- Added cors settings
- Linting setup
- Test setup

### CODE REVIEW

File: __test__/routes/index.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('GET /api/v1', () => {
+	it('should return 200 OK', async () => {
+		const response = await request(app).get(baseRoute);
+		expect(response.statusCode).toBe(200);
+	});
+});
+
+afterAll(() => {
+	// You're my wonderwall
```
Comment (Mafusn) [inline __test__/routes/index.test.js:22]: Love it <3

**Review by MortenMunk — APPROVED** [review]
lgtm

**Review by Mafusn — APPROVED** [review]
LGTM

**Review by TheMOHD3886 — APPROVED** [review]
lgtm 👍

**Pull request merged**